### PR TITLE
News item for 2.4.2-RC2

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

**Comment by johanandren** [comment]
LGTM!

**Comment by 2m** [comment]
This is very much based on the RC1 announcement. :)

**Comment by johanandren** [comment]
That's why it looks so good! ;)

File: news/_posts/2016-02-02-akka-2.4.2-RC2-released.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+* significant performance improvement for HTTP handling, now reaching roughly 75% of Spray’s performance—this is not the end of the performance work, we have only just begun
+* replacement of all uses of the Unit type (represented as BoxedUnit in Java) with the more descriptive types akka.Done (for signaling successful completion) and akka.NotUsed (for materialization results of stages that do not produce a value)
+* usage of Java 8 types in the Java DSLs: java.util.Optional instead of scala.Option and java.util.concurrent.CompletionStage instead of scala.concurrent.Future
+
+Especially the second and third point mean that porting code from Streams & HTTP 2.0 to Akka 2.4.2 will require some mechanical source code changes, please refer to the [migration guide](http://doc.akka.io/docs/akka/2.4.2-RC1/scala/stream/migration-guide-2.0-2.4-scala.html) for the details.
+
+Closed issues since the RC1 can be found [here](https://github.com/akka/akka/issues?utf8=%E2%9C%93&q=is%3Aissue+milestone%3A2.4.2+closed%3A%3E%3D2016-01-26).
```
Comment (rkuhn) [inline news/_posts/2016-02-02-akka-2.4.2-RC2-released.md]: Please add a remark that there are known issues, linking to the 2.4.2 milestone’s open tickets.

**Comment by rkuhn** [comment]
LGTM otherwise

**Pull request merged**